src/test/ui/resolve/issue-14254.stderr

```diff
@@ -2,7 +2,7 @@ error[E0425]: cannot find function `baz` in this scope
   --> $DIR/issue-14254.rs:29:9
    |
 29 |         baz();
-   |         ^^^ did you mean `self.baz(...)`?
+   |         ^^^ help: did you mean `self.baz`
```

Comment:
Are the "did you mean" questions... rhetoric?
Is it ok to omit question marks in this context, linguistically?